src/components/RessourceMenu/RessourceMenu.tsx

```diff
@@ -1,47 +1,74 @@
 import { useStore } from '../../store/store';
+
 import './RessourceMenu.scss';
 
+
+
+//const test = Math.floor(ressources.gold)
+
 export const RessourceMenu = () => {
   const ressources = useStore((state) => ({ ...state.ressources }));
   const buildingOutputs = useStore((state) => state.buildingOutputs);
   const reset = useStore((state) => state.reset);
 
+
+  const valueSmoothener: (m: number) => number = function (
+    m): number {
+
+    return (Math.round((m * 6000) / 100))
+  }
+
+
+  const remainingRessources: (v: number, t: number) => any = function (
+    v: number,
+    t: number,
+
+  ): any {
```

Comment:
I think the return type here could be string instead of any

Reply:
Thank you for your feedback 🥇